ros_buildfarm/templates/devel/devel_task.Dockerfile.em

```diff
@@ -11,8 +11,13 @@ MAINTAINER Dirk Thomas dthomas+buildfarm@@osrfoundation.org
 VOLUME ["/var/cache/apt/archives"]
 
 ENV DEBIAN_FRONTEND noninteractive
-RUN locale-gen en_US.UTF-8
-ENV LANG en_US.UTF-8
+
+@(TEMPLATE(
+    'snippet/setup_locale.Dockerfile.em',
+    os_name=os_name,
+    os_code_name=os_code_name,
+    timezone=timezone if timezone else None,
```

Comment:
I can't test it since it's available in all templates [here](https://github.com/ros-infrastructure/ros_buildfarm/blob/master/ros_buildfarm/templates/__init__.py#L76) i'll remove the conditional loading since you're right this won't work with timezone unset.